scripts/network/ping/baremetal_user_data.py

```diff
@@ -15,82 +17,75 @@
 # specific language governing permissions and limitations
 # under the License.
 
-'''
-Created on Jul 2, 2012
+# Created on Jul 2, 2012
+# @author: frank
```

Comment:
Remove the created/author comments.

Reply:
@rhtyd Those author comments were there previously. They're now gone :)